CRM/Admin/Form/OptionGroup.php

```diff
@@ -71,6 +71,8 @@ public function buildQuickForm() {
       CRM_Core_DAO::getAttribute('CRM_Core_DAO_OptionGroup', 'description')
     );
 
+    $this->addSelect('Data Type', CRM_Utils_Type::dataTypes(), TRUE);
```

Comment:
This doesn't let Option Group Form to load. Form to add option group results into a fatal error.

>Cannot determine default entity. CRM_Admin_Form_OptionGroup should implement getDefaultEntity().